modules/migration-using-must-gather.adoc

```diff
@@ -75,38 +78,52 @@ $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-g
+* To review the supported flags for the `oc adm must-gather` command, run the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather -h
+----
+
+* To use the default configuration of the `must-gather` tool, run either one of the following commands:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather --image={must-gather-v1-3}
 ----
 +
-The data is saved as `must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
-
-* Essential `must-gather` data collection, without Prometheus metrics, for a specific time duration:
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather
+----
++
+* To use the timeout flag with the `must-gather` tool, run the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather --image={must-gather-v1-3} \
-  -- /usr/bin/gather_<time>_essential <1>
+$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather --request-timeout 1m # <1>
 ----
-<1> Specify the time in hours. Allowed values are `1h`, `6h`, `24h`, `72h`, or `all`, for example, `gather_1h_essential` or `gather_all_essential`.
-
-* `must-gather` data collection with timeout:
+<1> In this example, the timeout is 1 minute.
+* To use the insecure TLS connection flag with the `must-gather` tool, run the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather --image={must-gather-v1-3} \
-  -- /usr/bin/gather_with_timeout <timeout> <1>
+$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather --skip-tls
 ----
-<1> Specify a timeout value in seconds.
-
-* Prometheus metrics data dump:
+* To use a combination of the insecure TLS connection, and the timeout flags with the `must-gather` tool, run the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather_metrics_dump
+$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather --request-timeout 15s --skip-tls #<1>
 ----
-This operation can take a long time. The data is saved as `must-gather/metrics/prom_data.tar.gz`.
+<1> In this example, the timeout is 15 seconds. By default, the `--skip-tls` flag value is `false`. Set the value to `true` to allow insecure TLS connections.
+
+.Verification
+
+. Verify that the Markdown output file is generated at the following location: `must-gather.local.89...054550/registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.5-sha256-0...84/clusters/a4...86/oadp-must-gather-summary.md`
+
+. Review the `must-gather` data in the Markdown file by opening the file in a Markdown previewer. For an example output, refer to the following image. You can upload this output file to a support case on the link:https://access.redhat.com/[Red{nbsp}Hat Customer Portal].
+
```

Comment:
Since the image below is introduced by this step, I would add a + here to make sure the image is indented under the step:
```suggestion
+
```